client/my-sites/theme/main.jsx

```diff
@@ -101,6 +103,34 @@ import ThemeStyleVariations from './theme-style-variations';
 
 import './style.scss';
 
+const LivePreviewButton = ( { siteSlug, themeId, themeType, isFullSiteEditingTheme } ) => {
+	if ( ! isFullSiteEditingTheme ) {
+		return null;
+	}
+
+	let themePath = '';
+	switch ( themeType ) {
+		case 'free':
+			themePath = 'pub/';
+			break;
+		case 'premium':
+			themePath = 'premium/';
+			break;
+	}
+
+	if ( themePath === '' ) {
+		return null;
+	}
+
+	return (
+		<Button
+			href={ `https://${ siteSlug }/wp-admin/site-editor.php?theme_preview=${ themePath }${ themeId }` }
```

Comment:
We can simply use the `stylesheet` of the theme. Is there any issue to use it directly?

Reply:
Addressed it in https://github.com/Automattic/wp-calypso/pull/78456/commits/b85547c4aef88e7fd9b7c3d67f4144ed4faf92ba. I was missing it! Thank you!